entrypoint.sh

```diff
@@ -6,4 +6,5 @@ if [ -f "requirements.txt" ]; then
   pip3 install -r requirements.txt
 fi
 
+git config --global --add safe.directory /github/workspace
```

Comment:
What is this being set for?

Reply:
Sorry for the incredibly slow reply, busy at work and life. Modern versions of Git require explicit knowledge of "safe" directories due to a CVE. Given this is our own workspace, this should be fine. https://github.blog/2022-04-12-git-security-vulnerability-announced/#cve-2022-24765